src/main/index.ts

```diff
@@ -59,6 +59,24 @@ import { UserStore } from "../common/user-store";
 import { WeblinkStore } from "../common/weblink-store";
 import { ExtensionsStore } from "../extensions/extensions-store";
 import { FilesystemProvisionerStore } from "./extension-filesystem";
+import { CaptureConsole, Dedupe, Offline } from "@sentry/integrations";
+
+// Initializing Sentry
+const Sentry =
+  // prevent `TypeError: mod.require is not a function`
+  // see https://docs.sentry.io/platforms/javascript/guides/electron/#webpack-configuration
+  process.type === "main"
```

Comment:
why do we need this if we are in `main/index.ts`. Can we just do `require("@sentry/electron/dist/main")` here

Reply:
I think you are right

Reply:
Fixed by @aleksfront https://github.com/lensapp/lens/pull/3252/commits/10025ba07f33a6e8ef34e4f092c38bf12a82836b